modules/files.nix

```
@@ -343,8 +343,13 @@ in
           # We therefore simply log the conflict and otherwise ignore it, mainly
           # to make the `files-target-config` test work as expected.
           if [[ -e "$realOut/$relTarget" ]]; then
-            echo "File conflict for file '$relTarget'" >&2
-            return
+            if [[ -d "$realOut/$relTarget" && $(realpath "$realOut/$relTarget") == $realOut && $recursive ]]; then
+                # exception: allow a "collision" for a recureively linked home *directory*
+                :
+            else
+                echo "File conflict for file '$relTarget'" >&2
+                return
```

```suggestion
              :
            else
              echo "File conflict for file '$relTarget'" >&2
              return
```

Indent with 2 spaces like the rest of the file.

---

I think the code style for nixos uses 4-space indents for bash code and 2-space indents for nix code.
…though now that I think about it, what is in the current code of home-manager is not up to style?
hm…
well, at least the file would be uniform.